shared/settings/account/add-modals.tsx

```diff
@@ -166,8 +166,10 @@ export const Phone = () => {
         style={styles.body}
       >
         <EnterPhoneNumberBody
-          onChangeNumber={onChangeNumber}
-          onChangeValidity={onChangeValidity}
+          onChangeNumber={(phoneNumber: string, validity: boolean) => {
```

Comment:
Best to do
```
const onChangeNumberCb = React.useCallback((phoneNumber: string, validity: boolean) => {
  onChangeNumber(phoneNumber)
  onChangeValidity(validity)
}, [])
```
above and pass that in here. Same for the rest of these.

Reply:
Done.